### PR TITLE
module and form icons

### PR DESCRIPTION
**Pull request by wpride**
Enable FormPlayer to differentiate form vs module commands as on mobile

cross https://github.com/dimagi/formplayer/pull/43

screen shot 2016-05-26 at 4 35 04 pm

### CODE REVIEW

File: corehq/apps/cloudcare/templates/formplayer/menu_list.html [hunk 1]
```diff
@@ -37,7 +37,11 @@
         <% if (imageUrl) { %>
         <img class="module-icon" src="<%- imageUrl %>"/>
         <% } else { %>
-        <i class="fa fa-pencil fa-4x"></i>
+            <% if (navState === "JUMP") { %>
+                <i class="fa fa-pencil fa-4x"></i>
```
Comment (benrudolph) [inline corehq/apps/cloudcare/templates/formplayer/menu_list.html]: nice i didn't even know about `fa-4x`!

**Comment by czue** [comment]
is that first column in the table just to hold the icon? looks kind of odd with the extra width and icon off to one side. not blocking now but guessing would be good to clean up before launch

**Comment by czue** [comment]
(looks nice though)

**Comment by benrudolph** [comment]
agree with that, a `text-center` class on the parent element may help make it a bit more visually appealing

**Comment by benrudolph** [comment]
or you could just not even make it a column and just have it sit right besides the text

**Comment by benrudolph** [comment]
(not going to block on that question though)

**Comment by benrudolph** [comment]
ah shit this wasn't going into master

**Pull request merged**